scripts/sync_packages.sh

```diff
@@ -1,35 +1,41 @@
 #!/bin/bash
-robot_name="${HOSTNAME//-b1}"
```

Comment:
please use robot_name and not the variable $ROBOT.  I can't ensure that this variable is always right, for example the root user hasn't got a default value. This will be changed as soon as we have a new network configuration, we will use nmap to check which Pcs are connected to the robot intern network.